storage/2020-08-04/datalakestore/filesystems/README.md

```diff
@@ -15,11 +15,7 @@ import (
 	"context"
 	"fmt"
 	"os"
-    "time"
 	
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/adal"
-    "github.com/Azure/go-autorest/autorest/azure"
     "github.com/hashicorp/go-azure-helpers/authentication"
     "github.com/hashicorp/go-azure-helpers/sender"
```

Comment:
these two shouldn't be needed with `go-azure-sdk`, since auth & the base layer are located within `github.com/hashicorp/go-azure-sdk/sdk` - could we fix this one? (equally, we're not using ADAL auth anymore)

Reply:
should we remove the readme changes and fix them all at the end of the refactor?

Reply:
Sure that works, otherwise tbh it might be worth leaving this as-is and checking them at the end for now - whichever is easiest for you 👍